### ISSUE TITLE
Console error when interacting with a contract with duplicated function names

### ISSUE THREAD

**Issue by Pabl0cks** [BuidlGuidl/abi.ninja#124]
Only happening after #121, steps to reproduce:

- Load `0xfDc1306fEC0C26Ed7012F863983cC138b1Ccb528` on Sepolia (challenge 0 contract)
- Click on any function, Read `balanceOf` for example
- You get this error on the console, but you can interact normally with it

Error:
```
client.js:2 Warning: validateDOMNesting(...): <button> cannot appear as a descendant of <button>.
    at button
    at button
    at div
    at div
    at div
    at div
    at MethodSelector (webpack-internal:///./components/scaffold-eth/Contract/MethodSelector.tsx:15:11)
    at ul
    at div
    at div
    at ContractUI (webpack-internal:///./components/scaffold-eth/Contract/ContractUI.tsx:68:11)
    at div
    at div
    at ContractDetailPage (webpack-internal:///./pages/[contractAddress]/[network].tsx:46:11)
    at main
    at div
    at div
    at ModalProvider (webpack-internal:///./node_modules/@rainbow-me/rainbowkit/dist/index.js:6206:26)
    at ShowBalanceProvider (webpack-internal:///./node_modules/@rainbow-me/rainbowkit/dist/index.js:1008:32)
    at TransactionStoreProvider (webpack-internal:///./node_modules/@rainbow-me/rainbowkit/dist/index.js:1499:3)
    at ModalSizeProvider (webpack-internal:///./node_modules/@rainbow-me/rainbowkit/dist/index.js:1696:3)
    at I18nProvider (webpack-internal:///./node_modules/@rainbow-me/rainbowkit/dist/index.js:797:23)
    at WalletButtonProvider (webpack-internal:///./node_modules/@rainbow-me/rainbowkit/dist/index.js:1670:33)
    at RainbowKitChainProvider (webpack-internal:///./node_modules/@rainbow-me/rainbowkit/dist/index.js:970:3)
    at RainbowKitProvider (webpack-internal:///./node_modules/@rainbow-me/rainbowkit/dist/index.js:2508:3)
    at ScaffoldEthApp (webpack-internal:///./pages/_app.tsx:49:11)
    at QueryClientProvider (webpack-internal:///./node_modules/@tanstack/react-query/build/modern/QueryClientProvider.js:28:3)
    at Hydrate (webpack-internal:///./node_modules/wagmi/dist/esm/hydrate.js:11:13)
    at WagmiProvider (webpack-internal:///./node_modules/wagmi/dist/esm/context.js:13:13)
    at O (webpack-internal:///./node_modules/next-themes/dist/index.mjs:7:429)
    at z (webpack-internal:///./node_modules/next-themes/dist/index.mjs:7:310)
    at PlausibleProvider (webpack-internal:///./node_modules/next-plausible/dist/index.esm.js:149:13)
    at ScaffoldEthAppWithProviders
    at PathnameContextProviderAdapter (webpack-internal:///./node_modules/next/dist/shared/lib/router/adapters.js:74:11)
    at ErrorBoundary (webpack-internal:///./node_modules/next/dist/compiled/@next/react-dev-overlay/dist/client.js:303:63)
    at ReactDevOverlay (webpack-internal:///./node_modules/next/dist/compiled/@next/react-dev-overlay/dist/client.js:852:919)
    at Container (webpack-internal:///./node_modules/next/dist/client/index.js:77:1)
    at AppContainer (webpack-internal:///./node_modules/next/dist/client/index.js:181:11)
    at Root (webpack-internal:///./node_modules/next/dist/client/index.js:360:11)
```

**Comment by portdeveloper** [comment]
Oh sorry about this! This is really because of me nesting buttons and yeeting to prod! It has been on my list of todos for a while. Shiv fixed it in wagmi v2 migration tho!

**Comment by technophile-04** [comment]
Fixed at #121
